lib/bot.js

```diff
@@ -98,6 +100,24 @@ class Bot {
       this.sendToDiscord(author, to, `*${text}*`);
     });
 
+    this.ircClient.on('join', (channel, nick) => {
+      if (this.disableJoinPartMsgs) return;
+      if (nick === this.nickname && !this.announceSelfJoin) return;
```

Comment:
Indeed; in my edge case, I want to avoid the rare instances where users may start having one-sided conversations into the void. But, so far, that has yet to happen. So maybe it is not a big enough deal :)